third_party/python/cpplint/BUILD

```diff
@@ -2,8 +2,14 @@ licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
-pex_binary(
+load("@rules_python//python:defs.bzl", "py_binary")
+load("@heron_py_deps//:requirements.bzl", "requirement")
+
+py_binary(
     name = "cpplint",
-    entrypoint = "cpplint",
-    reqs = ["cpplint==1.3.0"],
+    srcs = ["empty.py"],
+    main = "cpplint",
```

Comment:
Could this be causing an infinite recursion/expansion of the build rule?

Reply:
I guess it could be possible. @nicknezis would have a better idea.   He's been our go to Bazel person,